### PR TITLE
Signals: Org-specific signals

### PR DESCRIPTION
**Pull request by bfilar**

### CODE REVIEW

File: signals/content/subject_contains_org_display_name.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: "Subject: Matches entry in the $org_display_names list"
+type: "query"
+source: |  
+  subject.subject in~ $org_display_names
```
Comment (morriscode) [inline signals/content/subject_contains_org_display_name.yml:4]:
```suggestion
  any($org_display_names, strings.icontains(subject.subject, . ))
```
The existing logic would have to be an exact char match (case insens). I'm guessing by the name that's what you want, but I'd wager a contains would be more helpful?

File: signals/headers/headers_large_num_unknown_recipients.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: "Headers: Large unknown recipient list"
+type: "query"
+source: |  
+  (
```
Comment (morriscode) [inline signals/headers/headers_large_num_unknown_recipients.yml:4]:
```suggestion
```
Don't think these are needed

File: signals/headers/headers_large_num_unknown_recipients.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    length(recipients.to) > 10
+    and length(filter(recipients.to,
+                      .email.domain.domain not in $org_domains
+                      and .email.email not in $recipient_emails
+               )
+    ) >= 10
+  )
```
Comment (morriscode) [inline signals/headers/headers_large_num_unknown_recipients.yml:11]:
```suggestion
```

**Review by morriscode — APPROVED** [review]

File: signals/content/subject_contains_org_display_name.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+name: "Subject: Matches entry in the $org_display_names list"
+type: "query"
+source: |  
+  any($org_display_names, strings.icontains(subject.subject, . ))
```
Comment (rw-access) [inline signals/content/subject_contains_org_display_name.yml:4]: This list is probably too large for this to be performant for `any` checks like this

Comment (rw-access) [reply signals/content/subject_contains_org_display_name.yml:4]: I would say stick with `in`, `in~` with large lists for now

Comment (bfilar) [reply signals/content/subject_contains_org_display_name.yml:4]: Awesome! Yeah, I wasn't sure the best way to do that

**Pull request merged**